chat/src/test/java/greencity/mapping/ChatMessageWithFileDtoMapperTest.java

```diff
@@ -24,13 +29,48 @@ void init() {
             new Participant(1L, "name", "asd@asd.asd", null,
                 null, UserStatus.ACTIVATED, Role.ROLE_USER, null),
             "content", null, null, "fileName", "AUDIO",
-            "https://example.wav");
+            "https://example.wav", null);
         expected = new ChatMessageWithFileDto(1L, 1L, 1L, "content",
-            null, "fileName", "AUDIO", "https://example.wav");
+            null, "fileName", "AUDIO", "https://example.wav", null);
     }
 
     @Test
     void convert() {
         assertEquals(expected, chatMessageWithFileDtoMapper.convert(chatMessage));
     }
+
+    @Test
+    void convertWithLikes() {
+        Set<Participant> likes = new HashSet<>();
+        likes.add(new Participant(2L, "user2", "user2@example.com", null,
+            null, UserStatus.ACTIVATED, Role.ROLE_USER, null));
+        chatMessage.setLikes(likes);
+
+        Set<UserDto> expectedLikes = new HashSet<>();
+        expectedLikes.add(new UserDto(2L, "user2", null));
+        expected.setLikes(expectedLikes);
+
+        assertEquals(expected, chatMessageWithFileDtoMapper.convert(chatMessage));
+    }
+
+    @Test
+    void convertWithEmptyLikes() {
+        chatMessage.setLikes(Collections.emptySet());
+        expected.setLikes(Collections.emptySet());
+
+        assertEquals(expected, chatMessageWithFileDtoMapper.convert(chatMessage));
+    }
+
+    @Test
+    void convertWithNullFields() {
+        chatMessage.setFileName(null);
+        chatMessage.setFileType(null);
+        chatMessage.setFileUrl(null);
+
+        expected.setFileName(null);
+        expected.setFileType(null);
+        expected.setFileUrl(null);
```

Comment:
Recommendation: do not modify many fields, just create expected in method with all required data. 
It also can lead to some problems when tests are launched in parallel.